docs/config-service.rst

```diff
@@ -48,22 +48,64 @@ Plugin Configuration
 ``[gcp.event_consumer]``
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-All configuration options above in the general ``[gcp]`` may be used here. There are no specific Google Pub/Sub Consumer-related configuration options.
+All configuration options above in the general ``[gcp]`` may be used here. Additional Google Pub/Sub Consumer-related configuration options are:
 
+.. option:: topic="STR"
+
+    `Required`: A topic to which the Event Consumer client must subscribe.
+
+    For more information on Google Pub/Sub topics, please see `Google's docs on managing topics <topics>`_.
+
+.. option:: subscription="STR"
+
+    `Required`: A subscription to the ``topic`` from which the Event Consumer client will pull.
+
+    For more information on Google Pub/Sub subscriptions, please see `Google's docs on managing subscriptions <subscriptions>`_.
 
 ``[gcp.enricher]``
 ~~~~~~~~~~~~~~~~~~
 
-All configuration options above in the general ``[gcp]`` may be used here. There are no specific Google Compute Engine Enricher-related configuration options.
+All configuration options above in the general ``[gcp]`` may be used here. Additional Google Compute Engine configuration options are:
+
+
+.. option:: dns_zone="STR"
+
+    `Required`: DNS zone to validate the correctness of A records before publishing. Must be a fully-qualified domain name (FQDN), ending in ``.``, e.g. ``example.com.``.
 
 
 ``[gcp.publisher]``
 ~~~~~~~~~~~~~~~~~~~
 
-All configuration options above in the general ``[gcp]`` may be used here. There are no specific Google DNS Publisher-related configuration options.
+All configuration options above in the general ``[gcp]`` may be used here. Additional Google Cloud DNS configuration options are:
+
+.. describe:: dns_zone="STR"
```

Comment:
Originally I had this as `.. option::` but Sphinx warns/fails because it's also defined under the `[gcp.enricher]` section. I thought about putting it under `[gcp]` but that may be a bit misleading since the event consumer doesn't use/need it. `.. describe::` looks the same when rendering so ¯\\_(ツ)_/¯ .